### PR TITLE
Update botometer assessment

### PR DESCRIPTION
**Pull request by michelbl**

### CODE REVIEW

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 1]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
...
 probably inauthentic         | 3.2–3.8 score
-perhaps *                    | 2.0–3.2 score
+perhaps                      | 2.0–3.2 score
 probably authentic           | 1.4–2.0 score
-apparently authentic          | 1.0–1.4 score
+apparently authentic         | 1.0–1.4 score
 undoubtedly authentic        | 0.0–1.0 score
 
-
Botometer scores of the classified accounts were then calculated on the 19th and 24th Apr 2019.
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]: Make it clear that the Botometer scores are based on data from the same time. Maybe remove any date specification at that stage and only describe the normalisation process later?

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 2]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
 Manual classification        | Estimated Botometer score range
 -----------------------------|--------------------------------
 undoubtedly inauthentic      | 4.2–5.0 score
-apparently inauthentic        | 3.8–4.2 score
+apparently inauthentic       | 3.8–4.2 score
 probably inauthentic         | 3.2–3.8 score
-perhaps *                    | 2.0–3.2 score
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]: > I had added an asterisk to show that it was both “perhaps authentic” and “perhaps inauthentic”.

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 3]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
...
 probably authentic           | 1.4–2.0 score
-apparently authentic          | 1.0–1.4 score
+apparently authentic         | 1.0–1.4 score
 undoubtedly authentic        | 0.0–1.0 score
 
-
Botometer scores of the classified accounts were then calculated on the 19th and 24th Apr 2019.
+Botometer scores of the classified accounts were then calculated on May 22nd, 2019.
+
+This original dataset is largely unbalanced in favour of suspicious accounts, so we added a hundred of accounts that we know are authentic (friends and celebrities).
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]: > was

?

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 4]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
...
 
 ###### Limitations
 {: .no_toc}
 
-This time difference in computation is a human error and problematic, as some accounts were not available anymore and differences in assessment can also be explained by the time gap (a seemingly inauthentic account in February might become more human-like two months later, and the other way around).
-
-The difference is thus necessarily overestimated.
+This time difference in computation is a human error and problematic, as some accounts were not available anymore and differences in assessment can also be induced by the time gap (a seemingly inauthentic account in February might become more human-like two months later, and the other way around). In order to mitigate this issue, we provided the Botometer API with timelines and mentions as close as possible to their value on March 1st, 2019. In particular, we discarded tweets newer than March 1st, 2019.
 
+This evaluation was made on a dataset that we built to be highly varied, however it cannot capture the diversity of both authentic and inauthetic accounts that can be found in the tweeter community.
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]:
```suggestion
This evaluation was made on a dataset that we built to be highly varied, however it cannot capture the full diversity of both authentic and inauthentic accounts that can be found in the Twitter community.
```

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 5]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
...
 
 ###### Results
 {: .no_toc}
 
-![Botometer assessment](/assets/img/botometer-validation.png)
+![Botometer assessment](/assets/img/botometer-scatterplot.png)
 
-Botometer seems to have more certainty than our human classifiers, however it also had [seven more weeks](#limitations) worth of data to assess the accounts. Due to this time difference in assessment, we cannot conclude on the global reliability of Botometer. We can however see that the classification of “undoubtedly inauthentic” accounts seems to be quite as accurately made by Botometer as it is by humans.
+The two main clusters of accounts that were evaluated both as authentic or both as inauthentic by humans and Botometer, indicated that Botometer generally makes correct guesses.
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]:
```suggestion
The two main clusters of accounts that were evaluated both as authentic or both as inauthentic by humans and Botometer, indicating that Botometer generally makes guesses as correct as humans, even in a non-English context.
```

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 6]
```diff
@@ -97,29 +97,30 @@ Annotations were then transformed into numbers for easier comparison with Botome
...
-
Botometer scores of the classified accounts were then calculated on the 19th and 24th Apr 2019.
+Botometer scores of the classified accounts were then calculated on May 22nd, 2019.
+
+This original dataset is largely unbalanced in favour of suspicious accounts, so we added a hundred of accounts that we know are authentic (friends and celebrities).
 
 ###### Limitations
 {: .no_toc}
 
-This time difference in computation is a human error and problematic, as some accounts were not available anymore and differences in assessment can also be explained by the time gap (a seemingly inauthentic account in February might become more human-like two months later, and the other way around).
```
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]: Can't we just say that the data was the one from March 1st? I'm not sure it is “problematic” anymore, is it?

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 3 above)
Comment (MattiSG) [inline _pages/encyclopedia/3-qualification/3-tools.md]: When was that addition made?

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 2 above)
Comment (michelbl) [reply _pages/encyclopedia/3-qualification/3-tools.md]: Ok but I cannot find the footnote.

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 6 above)
Comment (michelbl) [reply _pages/encyclopedia/3-qualification/3-tools.md]: `The two main clusters of accounts that were evaluated both as authentic or both as inauthentic by humans and Botometer, indicates that Botometer generally makes guesses similar to humans’, even in a non-English context.`

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 5 above)
Comment (michelbl) [reply _pages/encyclopedia/3-qualification/3-tools.md]: `The two main clusters of accounts that were evaluated both as authentic or both as inauthentic by humans and Botometer, indicates that Botometer generally makes guesses similar to humans’, even in a non-English context.` ?

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 6 above)
Comment (michelbl) [reply _pages/encyclopedia/3-qualification/3-tools.md]: I could not perfectly bend the time... some timelines are partial and I don't know if the mentions returned by a search 2 months later are the same. Also, I don't know exactly which features are used by Botometer (profile image, etc)

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 2 above)
Comment (MattiSG) [reply _pages/encyclopedia/3-qualification/3-tools.md]: Sorry it wasn't a footnote, it meant “both authentic and inauthentic”. Definitely not clear then 😉

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 5 above)
Comment (MattiSG) [reply _pages/encyclopedia/3-qualification/3-tools.md]: Ok without the apostrophe after “humans” 😉

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 3 above)
Comment (michelbl) [reply _pages/encyclopedia/3-qualification/3-tools.md]: added

**Pull request merged**